lib/elixir/lib/node.ex

```diff
@@ -257,6 +257,38 @@ defmodule Node do
     :erlang.spawn_link(node, module, fun, args)
   end
 
+  @doc """
+  Returns the PID of a new process started by the application of `fun` on `node`,
+  and a reference for a monitor created to the new process. Otherwise works
+  like `spawn/1`
```

Comment:
We need to keep the first sentence short, as that is shown as a summary in ExDoc. Can you please take a look?